aptos-move/framework/aptos-framework/sources/staking_proxy.spec.move

```diff
@@ -21,23 +24,60 @@ spec aptos_framework::staking_proxy {
 
     spec set_vesting_contract_operator(owner: &signer, old_operator: address, new_operator: address) {
         // TODO: Can't verify `update_voter` in while loop.
-        pragma aborts_if_is_partial;
+        pragma verify = false;
     }
 
     spec set_staking_contract_operator(owner: &signer, old_operator: address, new_operator: address) {
-        use aptos_std::simple_map;
-        use aptos_framework::staking_contract::{Store};
+        pragma verify = false;
```

Comment:
If it can be proved with verify_duration_estimate = 120, please remove it

Reply:
There is a vector_ref loop in `set_staking_contract_operator`, so we used the pragma = false.